djcroco/tests/tests.py

```diff
@@ -78,7 +59,7 @@ def test_document_name(self):
     def test_document_size(self):
         # Ensure correct size
         self.assertEqual(self.instance.document.size, 679)
-        self.assertEqual(self.instance.document.size_human, '679 bytes')
+        self.assertEqual(self.instance.document.size_human, u'679\xa0bytes')
```

Comment:
1) Because that's what comes back from `document.size_human`, sadly.

2) Good question - I don't know.  I'm not sure what that would do.  `size_human` is in `fields.py` so we have control over it if necessary:

```
@property
def size_human(self):
    return filesizeformat(self.attrs['size'])
```